apps/comps/components/positions-table/index.tsx

```diff
@@ -86,11 +86,13 @@ export const PositionsTable: React.FC<PositionsTableProps> = ({
           <div>
             <div className="text-primary-foreground text-sm">
               {row.original.isLong ? "LONG" : "SHORT"}{" "}
-              {formatAmount(row.original.size)} {row.original.asset}
+              {formatAmount(Number(row.original.positionSize))}{" "}
+              {row.original.asset}
             </div>
             <div className="text-secondary-foreground text-xs">
-              {row.original.leverage}x leverage •{" "}
-              {formatAmount(row.original.collateral)} collateral
+              {row.original.leverage ? Number(row.original.leverage) : 0}x
+              leverage • {formatAmount(Number(row.original.collateralAmount))}{" "}
```

Comment:
Not a problem with this pr, but it sucks to return all these properties as strings from the db. Could probably use bigint mode.  That's another conversation.